irohad/ametsuchi/impl/postgres_block_index.hpp

```diff
@@ -27,13 +27,26 @@
 
 #include "model/transaction.hpp"  // for model::Transaction::CommandsType
 
+namespace shared_model {
+  namespace interface {
+    class Command;
+  }
+}
 namespace iroha {
   namespace ametsuchi {
     class PostgresBlockIndex : public BlockIndex {
+      template <class T>
+      using w = shared_model::detail::PolymorphicWrapper<T>;
+
+      /// Type of command
+      using CommandType = w<shared_model::interface::Command>;
```

Comment:
This is already defined in `shared_model::interface::Transaction`, why define it again here?